src/orange/orangeinp/ObjectInterface.hh

```diff
@@ -43,6 +43,9 @@ class ObjectInterface
     //!@}
 
   public:
+    // Anchored default destructor
+    virtual ~ObjectInterface() = 0;
```

Comment:
Is the switch from `protected` to `public` needed to work-around the issue?

Reply:
The switch from nonvirtual to virtual is needed to anchor it; the only reason it was protected before was to prevent deletion-to-nonvirtual-base.